### ISSUE TITLE
blank screen Strapi

### ISSUE THREAD

**Issue by tadinski** [alexzaganelli/strapi-plugin-email-designer#59]
After installation I get a blank screen
Strapi versions tried: 3.6.2 and 3.6.8

Browser console log shows an error: Prism is not defined

<img width="1062" alt="Screenshot 2021-08-29 at 19 35 22" src="https://user-images.githubusercontent.com/6358371/131259896-55535273-a155-4e77-96c9-4d4ff45456fe.png">

**Comment by alexzaganelli** [comment]
Hi @tadinski 

I can't reproduce this error. Have you already tried to delete the `node_modules` folder and run again the command `yarn`?

**Comment by tadinski** [comment]
@alexzaganelli Yes, I tried it. Below is my package.json in case it can help reproduce the issue.


`{
  "name": "ua-backend",
  "private": true,
  "version": "0.1.0",
  "description": "A Strapi application",
  "scripts": {
    "develop": "strapi develop",
    "start": "strapi start",
    "build": "strapi build",
    "strapi": "strapi"
  },
  "devDependencies": {},
  "dependencies": {
    "knex": "0.21.18",
    "mysql": "2.18.1",
    "strapi": "3.6.8",
    "strapi-admin": "3.6.8",
    "strapi-connector-bookshelf": "3.6.8",
    "strapi-plugin-content-manager": "3.6.8",
    "strapi-plugin-content-type-builder": "3.6.8",
    "strapi-plugin-email": "3.6.8",
    "strapi-plugin-graphql": "3.6.8",
    "strapi-plugin-i18n": "3.6.8",
    "strapi-plugin-import-export-content": "^0.3.4",
    "strapi-plugin-upload": "3.6.8",
    "strapi-plugin-users-permissions": "3.6.8",
    "strapi-provider-email-nodemailer": "^3.6.8",
    "strapi-provider-upload-cloudinary": "^3.6.8",
    "strapi-utils": "3.6.8",
    "stripe": "^8.171.0"
  },
  "author": {
    "name": "Pickle Rick"
  },
  "strapi": {
    "uuid": "e0d172b1-ea47-48e2-91c0-a673a926e2d1"
  },
  "engines": {
    "node": ">=10.16.0 <=14.x.x",
    "npm": "^6.0.0"
  },
  "license": "MIT"
}
`

**Comment by alexzaganelli** [comment]
I'm sorry @tadinski ,
I've just followed the instructions described on the README file and everything seems to works like a charm on my env.

May you please try to create a new project from scratch as described on the README?

Thanks
